docs/en/advanced/Performance-Tuning.md

```diff
@@ -47,21 +47,21 @@ The following is a checklist to run through to address common problems when tuni
 1. Are there warnings or errors in the master or worker logs related to thread pool exhaustion?
 
    Alluxio clients maintain a connection to the master to avoid using a new connection each time.
-   Until a client shuts down, it will occupy a server thread even if it is not sending requests.
+   Each client will occupy a server thread while the RPC is pending.
    This may deplete the master's thread pool; its size can be increased by setting
    `alluxio.master.worker.threads.max`, which defaults to 1/3 of the system's max file descriptor limit.
```

Comment:
Is this still accurate?

Reply:
Looks like we just changed the default value. Updated.